terraform/mgmtcluster.tf

```diff
@@ -36,14 +40,27 @@ locals {
   clouds = yamldecode(file("mycloud.${var.cloud_provider}.yaml"))
 }
 
+data "openstack_images_image_ids_v2" "images" {
+  name = var.image
+  sort = "updated_at:desc"
+}
+
 resource "openstack_compute_instance_v2" "mgmtcluster_server" {
   name              = "${var.prefix}-mgmtcluster"
-  image_name        = var.image
+  # image_name        = var.image
   flavor_name       = var.kind_flavor
```

Comment:
I am wondering, what will happen in underlying openstack infra when flavor with the disk is defined here. I tried it with `kind_flavor = "SCS-2V:4:20"` and in the horizon I see that flavor with 20GB disk is used and also 30 GB volume attached on /dev/sda. inside mgmt VM I see:
```
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda1        29G  5.5G   24G  20% /
```

Reply:
The disk that comes with the flavor normally is just ignored then? OK, I did not know this ... but it's good, as it makes everything robust here.

Reply:
It looks like that, but I don't know what happens on the underlying OpenStack layer, specifically what happens with the disk specified in the flavour. If it is ignored, it is good but otherwise can have a problem, don't you think so too?

Reply:
Ok, I investigated a little bit and found this [flavors docs](https://docs.openstack.org/nova/zed/user/flavors.html#overview).
There is a **Root Disk GB** section which states that `When booting from a persistent volume it is not used.` so most likely we can believe that.

There is also mention of [os_compute_api:servers:create:zero_disk_flavor](https://docs.openstack.org/nova/zed/configuration/policy.html#nova) policy rule, which probably works:
```bash
$ openstack server create --flavor "SCS-2V:4" --image "ubuntu-capi-image-v1.25.11" --network "capi-net" test
Only volume-backed servers are allowed for flavors with zero disk. (HTTP 403)
```
The same message I get if I used it with this repository(main branch), the capo controller reports the same error.

Reply:
Great, so we can move forward!